.github/workflows/yaml-lint-checker.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: Yaml Lint Release Notes Pull Request
+on:
+  pull_request:
+    paths:
+      - releases/**/release-notes/**.yaml
+  # Allow manual triggering
+  workflow_dispatch: {}
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+jobs:
+  yaml-lint-checker:
+    name: Release Notes Check Yaml 👀
+    runs-on: ubuntu-22.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
cool done, do we normally just use latest than a specific version? Not clear because you mentioned using a specific commit on the